distributed/semaphore.py

```diff
@@ -0,0 +1,259 @@
+import uuid
+from collections import defaultdict, deque
+from functools import partial
+import asyncio
+import dask
+import tornado.locks
+import tornado.queues
```

Comment:
This doesn't appear to be used.  My guess is that you have some excess imports here.